### ISSUE TITLE
Should watching be recursive?

### ISSUE THREAD

**Issue by philgyford** [WnP/pysass#1]
The readme says "watch included directories and source directory recursively" but I can't get it to watch files in subdirectories; I have to add each subdirectory as a separate `--include-path`.

Looking at the code, and [the watchdog docs](https://pythonhosted.org/watchdog/quickstart.html), should [this line](https://github.com/WnP/pysass/blob/master/pysass.py#L48):

```python
observer.schedule(ScssHandler(argv), path=path)
```

be:

```python
observer.schedule(ScssHandler(argv), path=path, recursive=True)
```

?

(Otherwise, thanks so much for this - it's really useful!)

**Comment by WnP** [comment]
Hi @philgyford,

Thanks for reporting this issue, this issue has be fixed in `0.1.0` release.

**Comment by philgyford** [comment]
Thank you! That seems to do the trick.
